plugins/callback/elastic.py

```diff
@@ -374,10 +374,14 @@ def v2_playbook_on_handler_task_start(self, task):
         )
 
     def v2_runner_on_failed(self, result, ignore_errors=False):
-        self.errors += 1
+        if ignore_errors:
+            status = 'ok'
```

Comment:
Shouldn't this be "ignored"? At least that's what the standard callback reports for ignored failures.

Reply:
Good point.

Let me explain a bit the context of the status manipulation from ansible to the spans.

Span status is based on 4 different types:

```
    "outcome": {
      "description": "Outcome of the span: success, failure, or unknown. Outcome may be one of a limited set of permitted values describing the success or failure of the span. It can be used for calculating error rates for outgoing requests.",
      "type": [
        "null",
        "string"
      ],
      "enum": [
        "success",
        "failure",
        "unknown",
        null
      ]
```

The task status in ansible map the below span status:

- `failed` -> `failure`
- `ok` -> `success`
- `skipped` -> `unknown`
- `included` - `success`

I could include a new ansible status called `ignored` which matches `success` or `unknown`.

I'd prefer to use `unknown` since it looks more accurate based on the 4 span status that are available. 

Let me put some code in place and see if it makes more sense.

Reply:
I'll transition this PR to draft